package.json

```diff
@@ -3,8 +3,10 @@
   "version": "3.2.1",
   "description": "Some tools for docker",
   "main": "lib/docker-toolbelt.js",
+  "types": "lib/docker-toolbelt.d.ts",
   "scripts": {
-    "prepublish": "npm run test && coffee -c lib",
```

Comment:
This should be
```
"prepublish": "require-npm4-to-publish",

Reply:
Ah yeah I forgot how this whole process worked, will change.